docker-compose.yml

```diff
@@ -63,24 +63,12 @@ services:
       - mysql
       - elasticsearch
       - redis
-      - kumascript
```

Comment:
This *was* weird. `docker-compose` starts all things by default, no? And the `kumascript` service depends on the `web` one. 
Actually, I could see the point of going the other way around. That instead, we say that `web` depends on `kumascript` but that the `kumascript` service doesn't depend on `web`.